src/middlewares/authentication.ts

```diff
@@ -0,0 +1,23 @@
+import { NextFunction, Request, Response } from 'express'
+import jwt, { JwtPayload } from 'jsonwebtoken'
+
+const SECRET = process.env.JWT_SECRET
```

Comment:
```suggestion
import  {JWT_SECRET}  from "../config/constants"
```
If possible, we can move this into constants for easier access on  environment variables